en/release-notes/upgrade-guide-v3.0.md

```diff
@@ -11,37 +11,31 @@ In order to ensure the smooth transition of an existing __Cocos Creator 2.4__ pr
 
-Although **we do not recommend projects under development, especially projects that are about to go live, upgrade to 3.0**, we will still launch the 2.x resource import tool in the 3.0 official version. This tool will support the perfect import of old project resources and the auxiliary import of codes. Code-assisted import will convert **Javascript** into **TypeScript**, add component type declarations, attribute declarations and function declarations. The references of components in the scene will be preserved, and the code inside the function will be imported in the form of comments, which can reduce the developer's upgrade difficulty.
+Although **we do not recommend projects under development, especially projects that are about to go live, upgrade to 3.0**, we will still launch the 2.x resource import tool in the 3.0 official version. This tool will support the perfect import of old project resources and the auxiliary import of codes. Code-assisted import will convert **JavaScript** into **TypeScript**, and automatically add component type declarations, attribute declarations and function declarations. The references of components in the scene will be preserved, and the code inside the function will be imported in the form of comments, which can reduce the developer's upgrade difficulty.
 
-Users only need to click **File -> Import -> CocosCreator 2D Project** in the main menu, and a window for importing the plug-in will appear.
+Developers only need to click **File -> Import -> Cocos Creator 2D Project** in the main menu, and a window for importing the plug-in will appear.
 
-![image](https://user-images.githubusercontent.com/1503156/100599538-20b8c880-333b-11eb-9831-bf176730b777.png)
-![image](https://user-images.githubusercontent.com/1503156/100599556-26161300-333b-11eb-8b85-31e144300f73.png)
-![image](https://user-images.githubusercontent.com/1503156/100599545-23b3b900-333b-11eb-844a-876ad09fe7c6.png)
+![image](import-menu.png)
 
-Then click the button under the left picture and select the root directory of the __Cocos Creator__ 2D project. The plug-in will automatically traverse all the resources in the project and present it to the user. Users can check the resources to be imported by themselves, and click the import button in the right picture after selection. The import is complete.
+Then click the button in the left picture below and select the root directory of the __Cocos Creator 2D__ project. The plug-in will automatically traverse all the resources in the project and present it to the developer. Developers can check the resources to be imported by themselves, and click the __Import__ button in the right picture below after selection. The import is complete.
+
+![image](import.png)
 
 If an existing project needs to be upgraded for special reasons, and you encounter technical or workload difficulties, you can also contact [slackmoehrle@cocos2d-x.org](mailto:slackmoehrle@cocos2d-x.org) for assistance!
```

Comment:
If an existing project needs to be upgraded for special reasons, and you encounter technical or workload difficulties, you can also contact [slackmoehrle@cocos2d-x.org](mailto:slackmoehrle@cocos2d-x.org) for assistance!
-->
If an existing project needs to be upgraded under special circumstances, and technical or workload difficulties are encountered, please contact [slackmoehrle@cocos2d-x.org](mailto:slackmoehrle@cocos2d-x.org) for assistance!

Reply:
Please use our \@cocos.com email.